### DISCUSSION TITLE
Enabling 'id' Input in Forms Generated by CreateInferencer with Antd UI

### DISCUSSION THREAD

**Discussion by tkd-yas** [refinedev/refine#5326] [Q&A] [answered]
I'm using Antd for the UI in my application and CreateInferencer to generate forms. 
However, these forms currently lack an 'id' input option. 
I am seeking a method to enable the inclusion of 'id' inputs in these forms.

**Comment by BatuhanW** [comment, marked as answer]
Hello @tkd-yas, having an ID field isn't a common use-case, so inferencer doesn't generate that form field. 

You should copy/paste the generated inferencer code and you can add id input manually, just like other inputs.

**Reply by tkd-yas** [reply]
@BatuhanW 
thank you for your reply,
I understoodd inference purpose is generate skeleton for development.
I do try your advice, and keep enjoy develop with refine.
